addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/protocol/MilightDiscover.java

```diff
@@ -76,62 +84,62 @@ public MilightDiscover(DiscoverResult discoverResult, int resendTimeoutInMillis,
         this.discoverResult = discoverResult;
     }
 
+    /**
+     * Closes the socket and waits for the thread to shutdown.
+     * You cannot reuse this object after calling dispose.
```

Comment:
But now you document it as it is it can only be called once, but this is not what dispose is about. It's used to reset the state for example when the configuration was updated.